<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="player2">
          <property name="text">
           <string>Player2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="player1">
          <property name="text">
           <string>Player1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="player1Label">
          <property name="text">
           <string>Player1 time left</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="player1TimeLeft"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="player2Label">
          <property name="text">
           <string>Player2 time left</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="player2TimeLeft"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="setPlayTime">
          <property name="text">
           <string>SetPlayTime</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="setPlayTimeLineEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuShakkikello_sovellus">
    <property name="title">
     <string>Shakkikello sovellus</string>
    </property>
   </widget>
   <addaction name="menuShakkikello_sovellus"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
